Products/ZenModel/migrate/updateZeneventserverstdioLogPaths.py

```diff
@@ -0,0 +1,46 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import os
+import servicemigration as sm
+from Products.ZenUtils.Utils import zenPath
+from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+
+log = logging.getLogger("zen.migrate")
+sm.require("1.1.10")
+
+class UpdateZeneventserverstdioLogPaths(Migrate.Step):
```

Comment:
We've got several more of these migrations to do - in the future, consider combining the logfilter definition with any other updates to the LogConfig to help keep the number of migrations and tests to a minimum